src/comgr.cpp

```diff
@@ -37,11 +37,7 @@
 #include <miopen/solver/implicitgemm_util.hpp>
 #include <miopen/stringutils.hpp>
 
-#if !defined(_WIN32)
```

Comment:
If you would like to keep backward compatibility with HIP SDK 5.7, then 
```C++
#if !defined(_WIN32) && HIP_PACKAGE_VERSION_MAJOR < 6
```
would do the job.